core/vm/instructions.go

```diff
@@ -415,15 +415,12 @@ func opSstore(instr instruction, env Environment, context *Context, memory *Memo
 	env.State().SetState(context.Address(), loc, common.BigToHash(val))
 }
 
-func opJump(instr instruction, env Environment, context *Context, memory *Memory, stack *stack) {
-}
-func opJumpi(instr instruction, env Environment, context *Context, memory *Memory, stack *stack) {
-}
-func opJumpdest(instr instruction, env Environment, context *Context, memory *Memory, stack *stack) {
-}
+func opJump(instr instruction, env Environment, context *Context, memory *Memory, stack *stack)     {}
+func opJumpi(instr instruction, env Environment, context *Context, memory *Memory, stack *stack)    {}
+func opJumpdest(instr instruction, env Environment, context *Context, memory *Memory, stack *stack) {}
```

Comment:
Invalid opcode do not make the program invalid. You still have to run it fully (resulting in OOG) and since paths are variable we **need** to run it unless we can statically determine a certain path is invalid given certain input. Static analysis is still way off